commitizen/providers/scm_provider.py

```diff
@@ -31,6 +31,12 @@ class ScmProvider(VersionProvider):
         "$patch": r"(?P<patch>\d+)",
         "$prerelease": r"(?P<prerelease>\w+\d+)?",
         "$devrelease": r"(?P<devrelease>\.dev\d+)?",
+        "${version}": r"(?P<version>.+)",
+        "${major}": r"(?P<major>\d+)",
+        "${minor}": r"(?P<minor>\d+)",
+        "${patch}": r"(?P<patch>\d+)",
+        "${prerelease}": r"(?P<prerelease>\w+\d+)?",
+        "${devrelease}": r"(?P<devrelease>\.dev\d+)?",
```

Comment:
Very similar to the 2 previous factorizable `TAG_FORMAT_REGEXS` so I think it should be factorized too,